go.mod

```diff
@@ -71,6 +71,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
```

Comment:
Is there any reason to use v4? It seems good to me to match this version to that of CRDs and the controller.

Reply:
Done.